### ISSUE TITLE
dose the distributed graph library support metis partition

### ISSUE THREAD

**Issue by YuxinxinChen** [IntelligentSoftwareSystems/Galois#388]
Hi Galois Team,

Does the distributed graph library support metis partition?

Thanks!

Yuxin

**Comment by l-hoang** [comment]
No, it does not: METIS is an offline partitioning method, and DistGalois/CuSP does streaming partitioning. If
you're curious about the details, see this paper:
https://www.cs.utexas.edu/~loc/papers/cusp.pdf
